extension/_locales/en/messages.json

```diff
@@ -0,0 +1,82 @@
+{
+    "extensionName": {
+      "message": "Firefox Translations",
+      "description": "Name of the extension"
```

Comment:
Is the name of the add-on supposed to remain in English? What's the policy? If "Translations" should stay in English, please add that to the description.

Reply:
I think the name should stay the same. I will add it to the description (cc @andrenatal).

Reply:
Being fixed in https://github.com/mozilla/firefox-translations/pull/116